src/hooks/helper/claim.ts

```diff
@@ -0,0 +1,287 @@
+import { BatchTxs } from './index';
+import { getAddressEnum } from './../../store/dapp-staking/calculation';
+import { Struct, Option } from '@polkadot/types';
+import { ApiPromise } from '@polkadot/api';
+
+interface ContractEraStake extends Struct {
```

Comment:
This has been modified in the latest version:

https://www.notion.so/staketechnologies/dApps-Staking-Individual-Claim-67654302ddb843bc89f505358205fd88#322b0a67fad3466db6ee5e22db5c92ec

Reply:
Sorry, I don't get it. `ContractEraStake` interface is used for querying `contractEraStake`.
![image](https://user-images.githubusercontent.com/92044428/155935529-9677a012-de43-497b-a5dc-dd6dc30a9350.png)